### PR TITLE
Fix prepare mailer

### LINKED ISSUE #797: Mixin in mailer prepare is not included in mailers
Hi the following code does not work for me:

environment.rb:
```
  mailer do
    root 'lib/some-app/mailers'

    # See http://hanamirb.org/guides/mailers/delivery
    delivery :test

    # This should work
     prepare do
       include SomeAPP::Mailers::I18n
     end
  end
```

I can confirm that doing a direct include in a mailer works. For example:
```
module Mailers::Users
  class Welcome
    include Hanami::Mailer
    include SomeApp::Mailers::I18n
  end
end
```

### PR DESCRIPTION
**Pull request by AlfonsoUceda**
This PR fixes #797

### CODE REVIEW

**Comment by AlfonsoUceda** [comment]
cc @jodosha

File: spec/integration/mailer_spec.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+RSpec.describe "Mailer", type: :cli do
+  it "use a mailer" do
```
Comment (AlfonsoUceda) [inline spec/integration/mailer_spec.rb]: I think I need to amend the test's name.

**Comment by AlfonsoUceda** [comment]
cc @hanami/core

File: spec/integration/mailer_spec.rb (hunk 1 above)
Comment (cllns) [inline spec/integration/mailer_spec.rb]: tiny grammar change: should be `it "uses..."`

**Comment by jodosha** [comment]
@AlfonsoUceda Thank you. I've changed the target branch from `develop` to `master` and the milestone from `1.1.0` to `1.0.1` as we want to this fix to be shipped soon.

**Pull request merged**